src/style.css

```
@@ -0,0 +1,81 @@
+* {
+  margin: 0;
+}
+
+.container {
+  display: inline-block;
+}
+
+.star-row {
+  display: flex;
+  width: 100px;
+  align-items: center;
+}
```

```suggestion
.star-row {
  display: flex;
}
```
the same, these styles are not needed